docs/concepts/copilot/copilot-ai-capabilities.md

```diff
@@ -0,0 +1,169 @@
+  - generativeai
+  - capabilities
+---
+
+Prophecy Data Copilot provides suggestions from an AI model as you develop your data Pipelines and Models. You can view and incorporate suggestions directly within the Prophecy visual editor and code editor. Data Copilot makes suggestions for your entire Pipeline, for a single Gem (transformation), and even for individual expressions within each Gem.
+
+## Supported AI capabilities by engine
+
+Data Copilot supports the following capabilites for the Spark and SQL engines:
+
+| AI Capability                                      | Spark                    | SQL                      |
+| -------------------------------------------------- | ------------------------ | ------------------------ |
+| Start a new Pipeline (Spark) or Model (SQL)        | ![Tick](./img/tick.svg)  | ![Tick](./img/tick.svg)  |
+| Modify an existing Pipeline (Spark) or Model (SQL) | ![Tick](./img/cross.svg) | ![Tick](./img/tick.svg)  |
+| Suggest Gems                                       | ![Tick](./img/tick.svg)  | ![Tick](./img/tick.svg)  |
+| Suggest Expressions                                | ![Tick](./img/tick.svg)  | ![Tick](./img/tick.svg)  |
+| Generate with AI, Scripts                          | ![Tick](./img/tick.svg)  | ![Tick](./img/cross.svg) |
+| Generate with AI Functions                         | ![Tick](./img/tick.svg)  | ![Tick](./img/tick.svg)  |
+| Map with AI                                        | ![Tick](./img/cross.svg) | ![Tick](./img/tick.svg)  |
+| Code with AI                                       | ![Tick](./img/cross.svg) | ![Tick](./img/tick.svg)  |
+| Fix with AI, Gems                                  | ![Tick](./img/cross.svg) | ![Tick](./img/tick.svg)  |
+| Fix with AI, Expressions                           | ![Tick](./img/tick.svg)  | ![Tick](./img/tick.svg)  |
+| Auto Documentation                                 | ![Tick](./img/tick.svg)  | ![Tick](./img/tick.svg)  |
+| Data Tests and Quality Checks                      | ![Tick](./img/tick.svg)  | ![Tick](./img/tick.svg)  |
+
+## Text to Pipelines
+
+Get started on a new Pipeline quickly by typing your query into the text box and Data Copilot will generate a new Pipeline or modify an existing one.
+
+### Start a new Pipeline
+
+![Start a Pipeline](img/copilot_text_to_Pipeline.png)
+
+The following example uses Data Copilot to help start a Pipeline:
+
+1. Type a prompt with English text, such as `Which customers shipped the largest orders this year?`
+2. Data Copilot uses metadata from the accessible Datasets, Seeds, Pipelines, and Models, to create a Knowledge Graph.
+3. Data Copilot uses OpenAI's language model to create the Pipeline based on the text prompt, with the Knowledge Graph as the context. This Pipeline is accessible in the visual editor as well as the code editor.
+4. You can keep or reject the suggested Pipeline and interactively execute it to see the results.
+5. View Data Copilot's suggested changes in the visual editor.
```

Comment:
Review suggested changes - this step should be done before accepting/rejecting. 

#4 can become #5.

Reply:
And let's mention that review step is optional